castai/sdk/api.gen.go

```diff
@@ -101,20 +101,6 @@ const (
 	STATUSUNKNOWN  CastaiSsoV1beta1SSOConnectionStatus = "STATUS_UNKNOWN"
 )
 
-// Defines values for ExternalclusterV1ClusterReconcileInfoReconcileMode.
```

Comment:
Are the changes in this file expected, doesn't look related to the change?

Reply:
The PR fails because the SDK is not updated. It takes open api spec from the web so I had to re-generate it.